integration/src/integration/java/pl/allegro/tech/hermes/integration/KafkaSingleMessageReaderTest.java

```diff
@@ -68,8 +68,8 @@ public void shouldFetchSingleAvroMessage() throws IOException {
                 .withContentType(AVRO).build();
         operations.buildTopic(topic);
         wait.untilTopicDetailsAreCreated(topic.getName());
-        assertThat(publisher.publish(topic.getQualifiedName(), avroUser.asBytes())
-                .getStatus()).isEqualTo(CREATED.getStatusCode());
+
+        publisher.publish(topic.getQualifiedName(), avroUser.asBytes(), Response.Status.CREATED);
```

Comment:
I'd argue that
`import static pl.allegro.tech.hermes.integration.test.HermesAssertions.assertThat

assertThat(publisher.publish(topic.getQualifiedName(), avroUser.asBytes()).hasStatus(CREATED);`

would be more verbose and clear at this stage than having it tested inside the publisher.